cve_bin_tool/checkers/docker.py

```diff
@@ -0,0 +1,16 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+
+"""
+CVE checker for boinc
+https://www.cvedetails.com/product/28125/Docker-Docker.html?vendor_id=13534
+"""
+from cve_bin_tool.checkers import Checker
+
+
+class DockerChecker(Checker):
+    CONTAINS_PATTERNS = []
+    FILENAME_PATTERNS = []
+    VERSION_PATTERNS = [r"docker ([0-9]+\.[0-9]+\.[0-9]+)"]
```

Comment:
According to the strings in the tests it will be
```suggestion
    VERSION_PATTERNS = [r"docker/([0-9]+\.[0-9]+\.[0-9]+)"]
```